TELEMETRY.md

```diff
@@ -1,85 +1,332 @@
 # Telemetry sent by Addon
 
-
+PLEASE SEE THE JSON SCHEMA FOR DETAILED DESCRIPTION OF THE ADDON-SPECIFIC PING PAYLOAD: [https://github.com/biancadanforth/tracking-protection-shield-study/tree/master/schemas/schema.json](https://github.com/biancadanforth/tracking-protection-shield-study/tree/master/schemas/schema.json)
 
 ## Usual Firefox Telemetry is unaffected.
 
 - No change: `main` and other pings are UNAFFECTED by this addon.
 - Respects telemetry preferences.  If user has disabled telemetry, no telemetry will be sent.
 
 
-
 ## `shield-study` pings (common to all shield-studies)
 
 `shield-studies-addon-utils` sends the usual packets.
 
 The STUDY SPECIFIC ENDINGS this study supports are:
 
-- "voted",
-- "notification-x"
-- "window-or-fx-closed"
+- "expired"
+- "user-enabled-auto-private-browsing"
+- "user-disabled-builtin-tracking-protection"
+- "user-enabled-builtin-tracking-protection"
+- "introduction-confirmation-leave-study"
+- "page-action-confirmation-leave-study"
 
 
 ## `shield-study-addon` pings, specific to THIS study.
 
 Events instrumented in this study:
 
-- UI
-    - prompted (notification bar is shown)
+-Rreported UI events:
+    - "panel-shown"
+    - "panel-hidden"
+    - "panel-dismissed"
+
+- Reported user interactions:
+    - "page-action-clicked"
+    - "new-tab-closed"
+    - "introduction-accept"
+    - "introduction-reject"
+    - "introduction-confirmation-cancel"
+    - "introduction-confirmation-leave-study"
+    - "page-action-reject"
+    - "page-action-confirmation-cancel"
+    - "page-action-confirmation-leave-study"
+
+- Behavior Summary Stats
+    - panel opening time lengths + its median + its mean
+    - new tab opening time lengths + its median + its mean
+    - page action counters when page action button is clicked + its median + its mean
+    - number of overall page action badge clicks
 
-- Interactions
-    - voted
+- Reported covariate attributes
+    - is dnt enabled?
+    - is auto update enabled?
+    - is web history enabled?
+    - have any of the three most popular adblockers?
+    - user profile age in days
 
 
-## Example sequence for a 'voted => not sure' interaction
+## Example sequence for a user rejecting tracking protection via the introductory panel
 
 These are the `payload` fields from all pings in the `shield-study` and `shield-study-addon` buckets.
 
 ```
 
 // common fields
 
-branch        up-to-expectations-1        // should describe Question text
-study_name    57-perception-shield-study
-addon_version 1.0.0
+branch        private        // should describe Question text
+study_name    trackingProtectionMessagingExperiment
```

Comment:
This `study_name` will actually be one of: addon-tracking-protection-messaging-1433473-new-users or addon-tracking-protection-messaging-1433473-all-users per #157 . If you could update it, I'll make sure it gets merged today.